src/main/resources/mainnet.conf

```diff
@@ -33,9 +33,15 @@ scorex {
       "165.227.26.175:9030",
       "78.46.93.239:9030",
       "159.89.116.15:9030",
-      "83.212.114.255:9030"
+      "83.212.114.255:9030",
+      "136.244.110.145:9030",
+      "91.199.118.161:9030",
+      "104.186.109.55:9030",
+      "209.217.206.254:9030",
+      "145.239.4.27:9030",
+      "94.130.108.35:9030"
     ]
-    maxDeliveryChecks = 20
+    maxDeliveryChecks = 18
```

Comment:
Please add comment in the code on what is the rationale behind this value.

Reply:
Done in basic file (reference.conf). Copied to mainnet.conf also